controllers/suite_test.go

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	//+kubebuilder:scaffold:imports
+)
+
+//const controllerName = "ibmblockcsi-controller"
+
+// These tests use Ginkgo (BDD-style Go testing framework). Refer to
+// http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
+ 
+ var k8sClient client.Client
+ var testEnv *envtest.Environment
+ var kubeVersion = "1.18"
```

Comment:
this is doesn't really matter because it's only for [this](https://github.com/IBM/ibm-block-csi-operator/blob/d134fc86aae366543508d0396be4a055d40119b8/controllers/ibmblockcsi_controller.go#L205) line

Reply:
then could you change it to a more "dummy" value? e.g. `"dummyKubeVersion"`

Reply:
but our code is searching `kubeVersion` specificlly

Reply:
I'm referring to the string value, not the var name (that's why I used quotes)

Reply:
done